.editorconfig

```diff
@@ -21,6 +21,11 @@ indent_style = tab
 indent_style = space
 indent_size = 4
 
+# YAML does not support hard tabs.
+[docker-compose*.{yml,yaml}]
+indent_style = space
+indent_size = 2
```

Comment:
Might be best to just remove this addition and change the section above to 2 instead of 4 since that is the official tab value for YAML.